gnucash/gtkbuilder/dialog-preferences.glade

```diff
@@ -2578,14 +2578,98 @@ many months before the current month</property>
                     <property name="visible">True</property>
                     <property name="can-focus">False</property>
                     <property name="halign">start</property>
-                    <property name="label" translatable="yes">&lt;b&gt;Reconciling&lt;/b&gt;</property>
+                    <property name="label" translatable="yes">&lt;b&gt;"Jump" for multiple accounts&lt;/b&gt;</property>
```

Comment:
I am not keen on the use of quotes here, not sure what it should be, maybe...
`Jump action for multiple splits in basic view`

Reply:
I've reworded this to `Jump action for multiple splits` (it's not just the basic ledger, it applies to Jump with the transaction row selected in any register view mode).

Reply:
I see that now, not sure it occurred to me to try as splits were visible.